.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+| Q | A
+| --- | ---
+| Bug fix? | yes
+| New feature? | yes
+| BC breaks? | yes
+| Deprecations? | yes
+| Fixed tickets | fixes #issuenum
```

Comment:
Short and concise: "issuenum" can be replaced by double-click and entering/pasting the correct number.